src/averageAges.js

```diff
@@ -37,7 +50,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? filterByGender(people, 'f').filter((mother) =>
+      people.find((child) => child.mother === mother.name)
```

Comment:
you just need to evaluate, right - so receive a boolean output `.some` would do a faster job in this regard.
`.find` returns the entire object - but you don't need an object.